metrics/network/remote_network/remote-networking-cpu.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This script will measure CPU consumption while running an iperf3 bandwidth
+# network measurement using a remote setup, where host A will run a
+# server container and host B will run a client container.
+
+set -e
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/remote-networking-test-common.sh"
+
+# Set QEMU_PATH unless it's already set
+QEMU_PATH=${QEMU_PATH:-$(get_qemu_path)}
+# Time in which we sample the memory (seconds)
+middle_time=10
+# Time to run the server using iperf3 (seconds)
+server_time=30
+# Arguments to run the client
+extra_args="-d"
+
+# This function describes how to use this script
+function help {
+echo "$(cat << EOF
+Usage: $0 "[options]"
+	Description:
+		This script measures CPU consumption. Simultaneously, the
+		script runs a server container and host B runs a client
+		container.
+		To run this script, provide the following inputs:
+		- Interface name where swarm will run.
```

Comment:
Capitalize Swarm if you refer to the application. If you mean the noun, then add 'the' before it.